package.json

```diff
@@ -10,26 +10,25 @@
     "pretest": "npm run compile",
     "lint": "eslint src/*.js test-src/*.js",
     "lintfix": "eslint --fix src/*.js test-src/*.js",
-    "test": "npm run lint && npm run compile && NODE_ENV=test mocha test/*_test.js"
+    "test": "npm run lint && NODE_ENV=test mocha test/*_test.js",
```

Comment:
As i mentioned in the email, it's probably safe to remove the `NODE_ENV=test` section from here.